bridge/packages/feature/login/lib/src/login_view.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:core_designsystem/designsystem.dart';
+import 'package:core_model/model.dart';
+import 'package:feature_login/src/bloc/login_bloc.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:formz/formz.dart';
+
+class LoginView extends StatelessWidget {
+  const LoginView({
+    super.key,
+    required this.onNavBackClick,
+    required this.onNavHomeRequest,
+  });
+
+  final VoidCallback? onNavBackClick;
+  final VoidCallback? onNavHomeRequest;
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocListener<LoginBloc, LoginState>(
+      listener: (context, state) {
+        if (state.status.isFailure) {
+          ScaffoldMessenger.of(context)
+            ..hideCurrentSnackBar()
+            ..showSnackBar(
+              const SnackBar(content: Text('Authentication Failure')),
+            );
+        }
+
+        if (state.status.isSuccess) onNavHomeRequest?.call();
+      },
+      child: Column(
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              IconButton(
+                icon: const Icon(Icons.close),
+                onPressed: () {
+                  onNavBackClick?.call();
+                },
+              ),
+            ],
+          ),
+          const Spacer(),
+          _UserEmailInput(),
+          const SizedBox(
+            height: 32,
+            width: double.infinity,
+          ),
+          _UserPassword(),
+          const SizedBox(
+            height: 48,
```

Comment:
Try to avoid trailing commas if everything fits in a single line.

`const SizedBox(height: 48)` is much more concise

Reply:
Sometimes formatting lint breaks build because of missing commas 😬